### ISSUE TITLE
Devtools inspection is not available because it's in production mode or explicitly disabled by the author.

### ISSUE THREAD

**Issue by AndryRana** [vuejs/vue-devtools#1203]
### Version
5.3.3

### Browser and OS info
chrome Version 83.0.4103.61/mac os catalina10.15.4 

### Steps to reproduce
Devtools inspection is not available because it's in production mode or explicitly disabled by the author.

### What is expected?
Devtools inspection is not available because it's in production mode or explicitly disabled by the author.

### What is actually happening?
Devtools inspection is not available because it's in production mode or explicitly disabled by the author.

---
Devtools inspection is not available because it's in production mode or explicitly disabled by the author.

**Comment by thefirestrings666** [comment]
Yes I noticed it, very useful when running Firebase Emulators....

**Comment by WhyNotHugo** [comment]
I had the same issue with Vue2. I had to explicitly enable devtools with:

    Vue.config.devtools = true;

After upgrading to Vue3, I've no idea how to work around this any more.
